frontend/src/routes/Identities/Users/Users.tsx

```diff
@@ -1,10 +1,116 @@
+
+  const users = useMemo(() => rbacUsers?.sort((a, b) => compareStrings(a.metadata.name ?? '', b.metadata.name ?? '')) ?? [], [rbacUsers])
+
+  const keyFn = useCallback((user: RbacUser) => user.metadata.name ?? '', [])
+
+  const columns = useMemo<IAcmTableColumn<RbacUser>[]>(
+    () => [
+      {
+        header: t('Name'),
+        sort: 'metadata.name',
+        search: 'metadata.name',
+        // transforms: [cellWidth(40)],
+        cell: (user, search) => (
+          <span style={{ whiteSpace: 'nowrap' }}>
+            <Link to={generatePath(NavigationPath.identitiesUsersDetails, { id: user.metadata.uid ?? '' })}>
+              <HighlightSearchText text={user.metadata.name ?? ''} searchText={search} isTruncate />
+            </Link>
+          </span>
+        ),
+        exportContent: (user) => user.metadata.name ?? '',
+      },
+      {
+        header: t('Status'),
+        cell: () => <span>{t('Active')}</span>,
+        // transforms: [cellWidth(20)],
+        exportContent: () => 'Active',
+      },
+      {
+        header: t('Created'),
+        cell: (user) => {
+          return user.metadata.creationTimestamp ? (
+            <span style={{ whiteSpace: 'nowrap' }}>
+              <AcmTimestamp timestamp={user.metadata.creationTimestamp} />
+            </span>
+          ) : (
+            '-'
+          )
+        },
+        // transforms: [cellWidth(40)],
+        sort: 'metadata.creationTimestamp',
+        exportContent: (user) => user.metadata.creationTimestamp ? getISOStringTimestamp(user.metadata.creationTimestamp) : '',
+      },
+    ],
+    [t]
+  )
+
+  const rowActionResolver = useCallback(
+    (user: RbacUser): IAcmRowAction<RbacUser>[] => {
+      const actions: IAcmRowAction<RbacUser>[] = [
+        {
+          id: 'details',
+          title: (
+            <>
+              {t('Impersonate user')} <strong>{user.metadata.name}</strong>
+            </>
+          ),
+          click: () => {
+            navigate(generatePath(NavigationPath.identitiesUsersDetails, { id: user.metadata.uid ?? '' }))
+          },
+        },
+        {
+          id: 'edit',
+          title: t('Edit user'),
+          click: () => {
+            navigate(generatePath(NavigationPath.identitiesUsersDetails, { id: user.metadata.uid ?? '' }))
+          },
+        },
+        {
+          id: 'delete',
+          title: t('Delete user'),
+          click: () => {
+            navigate(generatePath(NavigationPath.identitiesUsersDetails, { id: user.metadata.uid ?? '' }))
+          },
+        },
+      ]
+
+      return actions
+    },
+    [t, navigate]
+  )
+
   return (
     <PageSection>
-      <div>Users list</div>
+      {loading ? (
+        <AcmLoadingPage />
+      ) : (
+        <AcmTable<RbacUser>
+          key="users-table"
+          columns={columns}
+          keyFn={keyFn}
+          items={users}
+          emptyState={<AcmEmptyState key="usersEmptyState" title={t("You don't have any users")} />}
```

Comment:
wdyt? or `There are no users` (`You` is the word that sounds weird here)
```suggestion
          emptyState={<AcmEmptyState key="usersEmptyState" title={t("No users")} />}
```